src/lab14.py

```diff
@@ -1540,11 +1537,13 @@ def speak_document(self):
         else:
             print(text)
 
+    @wbetools.js_hide
```

Comment:
Why should these be hidden?

Reply:
We never discuss them in the chapter. Perhaps that means we should just eliminate the code completely but I wasn't sure if it was useful in debugging so I left them in for now